gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/internal/nativeTargets.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.compose.internal
+
+import org.jetbrains.kotlin.konan.target.KonanTarget
+
+internal val SUPPORTED_NATIVE_TARGETS = setOf(
```

Comment:
Actually, Compose Compiler plugin supports more k/native targets than listed here. Compose Compile plugin can support any kotlin target which has compose-runtime compiled for it. Here is a list of compose-runtime targets: https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/compose/runtime/runtime/build.gradle#L75

`private val SUPPORTED_NATIVE_TARGETS = setOf(` - this is a recent change. I'll ask my team if linux,watchos and others were intentiomally excluded here.

Reply:
I see, please let me know. I can make the list match the list you linked, and add a comment there to remind that new targets should be added to this set.